src/main/java/com/minelittlepony/MineLittlePony.java

```diff
@@ -1,90 +1,67 @@
-import com.minelittlepony.renderer.RenderPonyZombie;
-import com.minelittlepony.renderer.RenderPonyZombieVillager;
+import com.minelittlepony.pony.data.IPonyData;
+import com.minelittlepony.pony.data.PonyDataSerialzier;
 import com.mumfrey.liteloader.core.LiteLoader;
-import com.mumfrey.liteloader.util.ModUtilities;
 import com.voxelmodpack.hdskins.HDSkinManager;
 import com.voxelmodpack.hdskins.gui.GuiSkins;
 import com.voxelmodpack.hdskins.skins.SkinServer;
 import net.minecraft.client.Minecraft;
-import net.minecraft.client.renderer.entity.Render;
 import net.minecraft.client.renderer.entity.RenderManager;
 import net.minecraft.client.resources.IReloadableResourceManager;
 import net.minecraft.client.resources.data.MetadataSerializer;
 import net.minecraft.client.settings.KeyBinding;
-import net.minecraft.entity.Entity;
-import net.minecraft.entity.monster.EntityEvoker;
-import net.minecraft.entity.monster.EntityHusk;
-import net.minecraft.entity.monster.EntityIllusionIllager;
-import net.minecraft.entity.monster.EntityPigZombie;
-import net.minecraft.entity.monster.EntitySkeleton;
-import net.minecraft.entity.monster.EntityStray;
-import net.minecraft.entity.monster.EntityVex;
-import net.minecraft.entity.monster.EntityVindicator;
-import net.minecraft.entity.monster.EntityWitherSkeleton;
-import net.minecraft.entity.monster.EntityZombie;
-import net.minecraft.entity.monster.EntityZombieVillager;
-import net.minecraft.entity.passive.EntityVillager;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.lwjgl.input.Keyboard;
 
-import java.util.Map;
-
+/**
+ * Static MineLittlePony singleton class. Everything's controlled from up here.
+ */
 public class MineLittlePony {
 
     public static final Logger logger = LogManager.getLogger("MineLittlePony");
 
-
     public static final String MOD_NAME = "Mine Little Pony";
     public static final String MOD_VERSION = "@VERSION@";
 
-    private static final String SKIN_SERVER_URL = "minelpskins.voxelmodpack.com";
+	private static final String SKIN_SERVER_URL = "minelpskins.voxelmodpack.com";
     private static final String GATEWAY_URL = "minelpskinmanager.voxelmodpack.com";
+
     private static final KeyBinding SETTINGS_GUI = new KeyBinding("Settings", Keyboard.KEY_F9, "Mine Little Pony");
 
     private static MineLittlePony instance;
 
-    private PonyConfig config;
-    private PonyManager ponyManager;
+    private final PonyConfig config;
+    private final PonyManager ponyManager;
 
-    private Map<Class<? extends Entity>, Render<?>> renderMap = Maps.newHashMap();
+    private final PonyRenderManager renderManager;
 
     MineLittlePony() {
         instance = this;
-    }
 
-    void init() {
         LiteLoader.getInput().registerKeyBinding(SETTINGS_GUI);
 
-        this.config = new PonyConfig();
-        this.ponyManager = new PonyManager(config);
+        config = new PonyConfig();
+        ponyManager = new PonyManager(config);
```

Comment:
You can cut down on the size of the commit by not formatting things you didn't edit. For the most part, try to use `this` where possible. It makes it more obvious on what variable is being changed or accessed.

Reply:
Normally I wouldn't, but in this case the entire codebase _desperately_ needed it. As in, I'm entirely convinced you started by decompiling the entire mod and pushing the result with no changes.

As for the use of `this`. I only ever use it where _absolutely_ necessary, which, ins most cases, it's not. Adding it creates extra noise, and anyone working on the code in any meaningful way is expected to have a development environment with suitable syntax highlighting installed.